docs/testing.md

```diff
@@ -0,0 +1,38 @@
+# Testing Open Angel Arena
+This is the continuation of [Install guide](/docs/install.md)
+Here is a guide how you can modify and run the OAA
+
+## Preparation
+You first need to open dev tools and load OAA project
+Also dont forget `node.js command prompt` from previous guide and ensure it is in `Open Angel Arena` directory
+
+## Building a game
+When you have just downloaded the sources you need to build evrything before running
+To begin with you need your command prompt, where you should type
+```
+dota 2 beta\addons\oaa> gulp
+```
+this part builds npcs and custom scripts.
+`You should rerun it every time you want to realunch OAA with new settings/files/localizations`
```

Comment:
this is no longer needed, and was part of building npc files (not localizations)